README.md

```diff
@@ -39,20 +39,29 @@ Install it with:
 
     cabal update && cabal install --ignore-project doctest
 
-Make sure that Cabal's `bindir` is on your `PATH`.
+Make sure that Cabal's `installdir` is on your `PATH`.
 
-On Linux:
+On Linux / macOS / BSD:
 
-    export PATH="$(cabal -v0 path --installdir):$PATH"
-
-On Mac OS X:
-
-    export PATH="$(cabal -v0 path --installdir):$PATH"
+```bash
+ # requires cabal-install 1.12, or later
+export PATH="$(cabal -v0 path --installdir):$PATH"
+```
+or
+```bash
+export PATH="$HOME/.local/bin:$PATH"
+```
 
 On Windows:
 
-    set PATH="%AppData%\cabal\bin\;%PATH%"
-
+```batch
+: requires cabal-install 1.12, or later
+for /f %i in ('cabal path -v0 --installdir') do set PATH=%i;%PATH%
```

Comment:
@mpilgrem The Internet™ says that this is apparently how to do things with `cmd.exe`.

1. Can you try if this works?
2. I kind of assume that this actually should be `for /f %i in ('cabal path -v0 --installdir') do set PATH="%i;%PATH%"` instead (note the quotes).  Does `%i` still get properly expended with the quotes?
3. Is the for-loop really required? Would something like `set PATH=('cabal -v0 --installdir');%PATH%` work instead?

Reply:
I am relying on [this documentation](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/for) about Command Prompt's `for`. It seems to me:
1.  the command whose output is being parsed needs to be between back-quotes and, I think, the use of the parsing key word `usebackq`;
2.  you need to be clear if this advice is at the command line or in a batch file: the former uses `%i`, the latter `%%i`;
3.  it is possible for a Windows path to contain spaces. I think that means you need the parsing key word `tokens=*` and the quotation marks around `%i;%PATH%`.

So, at the command line, I think it needs to be as follows:
~~~text
for /f "usebackq tokens=*" %i in (`cabal path -v0 --installdir`) do set PATH="%i;%PATH%"
~~~

but I have yet to test that.

Reply:
I was wrong about quotation marks around `%i;%PATH%`. I think this works at the command line
~~~text
for /f "usebackq tokens=*" %i in (`cabal path -v0 --installdir`) do set PATH=%i;%PATH%
~~~